core/lib/spree/core/search/product_filter.rb

```diff
@@ -0,0 +1,27 @@
+module Spree
+  module Core
+    module Search
+      class ProductFilter
+        attr_reader :filters
+
+        def initialize(taxon = nil)
+          @filters = []
+
+          if taxon.nil?
+            @filters << Spree::Core::ProductFilters.all_taxons
+          else
+            # @filters << Spree::Core::ProductFilters.taxons_below(taxon)
+            ## unless it's a root taxon? left open for demo purposes
+
+            @filters << Spree::Core::ProductFilters.price_filter
+            @filters << Spree::Core::ProductFilters.brand_filter
+          end
+        end
+
+        def self.partial_path
```

Comment:
@cbrunsdon you're right, it could be an awesome feature.

Recently I needed something like this for a store that had two "filter areas", one at the top and one on the side (each with its own set of models to search on) and I think using an instance method would have saved me a lot of time.